README.md

```diff
@@ -62,22 +62,24 @@ Goad will read your credentials from `~/.aws/credentials` or from the `AWS_ACCES
 
 ```sh
 # Get help:
-$ goad -h
-Usage of goad:
-  -c uint
-      number of concurrent requests (default 10)
-  -m string
-      HTTP method (default "GET")
-  -n uint
-      number of total requests to make (default 1000)
-  -r string
-      AWS regions to run in (comma separated, no spaces) (default "us-east-1")
-  -t uint
-      request timeout in seconds (default 15)
-  -u string
-      URL to load test (required)
-  -H string
-      HTTP Header to add to the request. This option can be set multiple times.
+$ goad --help
+usage: goad --url=URL [<flags>]
+
+A command-line chat application.
```

Comment:
I fixed the copy-pasta.

The idea is to keep both the current short flags and introduce clearer, more verbose long flags for clarity.